examples/worldview/src/scenes/sftrees/index.js

```diff
@@ -35,7 +35,7 @@ export const useScene = () => {
     // layerKeyframes: [],
     // tripKeyframe: {},
     cameraKeyframe: {
-      timings: [500, 3500],
+      timings: [500, 2500],
```

Comment:
what's the reason for this change?

Reply:
Don't need to wait an extra second anymore when testing the animation.